Framework/src/TaskRunner.cxx

```diff
@@ -413,14 +409,14 @@ void TaskRunner::startOfActivity()
   mTimerTotalDurationActivity.reset();
   mTotalNumberObjectsPublished = 0;
 
-  // We take the run number as set from the FairMQ options if it is there, otherwise the one from the config file
-  int run = mRunNumber > 0 ? mRunNumber : mConfigFile->get<int>("qc.config.Activity.number");
-  Activity activity(run,
+  // Start activity in module's stask and update objectsManager
+  Activity activity(mRunNumber,
                     mConfigFile->get<int>("qc.config.Activity.type"));
   ILOG(Info, Ops) << "Starting run " << mRunNumber << ENDM;
-  mCollector->setRunNumber(run);
+  mCollector->setRunNumber(mRunNumber);
```

Comment:
I don't see a `start()` method which would compute the run number, similarly to `CheckRunner` and `AggregatorRunner`. How is it set for `TaskRunner`?

Reply:
It is there but I extracted the computation of the run number to `computeRunNumber`.

Reply:
Yes, but I can't find any usage of `computeRunNumber` in `TaskRunner`. Maybe I'm blind though

Reply:
There was already a computeRunNumber in TaskRunner from when I did my tests to get the run number. 
I actally forgot to use the runnerUtils's method so I modified the PR along this line. 
Thanks

Reply:
ah, I see, this is why i couldn't find it in the diff.